package.json

```diff
@@ -28,11 +28,11 @@
   },
   "dependencies": {
     "passport": "^0.3.2",
-    "underscore": "^1.8.2",
-    "eslint": "^2.5.3",
-    "eslint-config-loopback": "^1.0.0"
+    "underscore": "^1.8.2"
   },
   "devDependencies": {
+    "eslint": "^2.5.3",
```

Comment:
> If you still need ESLint to run on Node.js < 4, then we recommend staying with ESLint v2.13.1 until you are ready to upgrade your Node.js version.

We should probably change package.json to specifically use this version and only get backwards compat/bug fixes. @bajtos Thoughts? Maybe update KB?